packages/cli/index.ts

```diff
@@ -20,6 +20,7 @@ import pkg from "./package.json" with { type: "json" };
 import { isBinaryFile } from "isbinaryfile";
 import { readPackageJSON, writePackageJSON } from "pkg-types";
 import { createDefaultTemplate } from "./template";
+import { detectPackageManager } from "../backend/server/utils/markdown";
```

Comment:
why is it in markdown utils? I don't think it should be there, let's have it inlined in this file!